boards/riscv/esp32c3_devkitm/esp32c3_devkitm.yaml

```diff
@@ -3,4 +3,4 @@ name: ESP32-C3
 type: mcu
 arch: riscv
 toolchain:
-  - espressif
+  - zephyr
```

Comment:
@stephanosio Can we proceed merging this to have new toolchain available and then we handle CI stuff in a dedicated matter? I can restore this entry to `espressif` and modify the documentation of make sure `export ZEPHYR_TOOLCHAIN_VARIANT="zephyr"` is kept. What do you think?